src/main/java/org/example/Pitcher.java

```diff
@@ -0,0 +1,11 @@
+package org.example;
+
+public interface Pitcher {
+
+  public String Pitching(String name);
```

Comment:
```suggestion
  public String pitching(String name);
```
Javaではメソッド名は小文字始まりのキャメルケースで定義します。
他メソッドも直しましょう。

Reply:
すべてのメソッドを小文字始まりのキャメルケースに修正しました。
ご確認の程よろしくお願い致します。